website/source/docs/config/index.html.md

```diff
@@ -91,9 +91,11 @@ sudo setcap cap_ipc_lock=+ep $(readlink -f $(which vault))
 
 ## Listener Reference
 
-For the `listener` section, the only supported listener currently
-is "tcp". Regardless of future plans, this is the recommended listener,
-since it allows for HA mode.
+For the `listener` section, the only required listener is "tcp".
+Regardless of future plans, this is the recommended listener,
+as it allows for HA mode. If you wish to use the Vault
+Enterprise interface in HashiCorp Atlas, you may add an "atlas" listener block
```

Comment:
can we link `"atlas" listener block` to the "connecting to vault enterprise..." section?